service_container/tags.rst

```diff
@@ -595,6 +595,9 @@ Tagged Services with Priority
 The tagged services can be prioritized using the ``priority`` attribute,
 thus providing a way to inject a sorted collection of services:
 
+.. tip::
+The priority is a positive or negative integer. The higher the number, the earlier the tagged service will be located in the collection.
+
```

Comment:
```suggestion
The tagged services can be prioritized using the ``priority`` attribute.
The priority is a positive or negative integer. The higher the number,
the earlier the tagged service will be located in the collection:
```

I think we can make this part of the main text, giving it more visibility and a smoother reading experience.